### PR TITLE
switch to react custom scrollbars

### LINKED ISSUE #670: switcht to react-custom-scrollbars
### Describe the feature

we should switch from simplebar-react to react-custom-scrollbars

### Why is this feature needed?

This feature is needed because of some problems occurred by simplar-react

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by florianjs**
## Which issues does this pull request close?
closes #670

### CODE REVIEW

File: lib/components/Table.tsx [hunk 1]
```diff
@@ -232,7 +231,6 @@ export const Table = <T, >({
   return (
     <div className={tw('flex flex-col gap-y-4 overflow-hidden')}>
       <div>
-        <SimpleBarReact>
```
Comment (DasProffi) [inline lib/components/Table.tsx:235]: Why did you remove the Scrollbar wrapper and not replace it? Does the new scrollbar appear automatically?

**Review by DasProffi — CHANGES_REQUESTED** [review]

**Comment by DasProffi** [comment]
The scrollbar for the table causes layout problems and for some reason the members table is missing completely
image

**Comment by florianjs** [comment]
> The scrollbar for the table causes layout problems and for some reason the members table is missing completely image

Thanks, the problem is if enable autoheigth without a min height, then the whole table is scrollable

File: lib/components/Table.tsx (hunk 1 above)
Comment (florianjs) [reply lib/components/Table.tsx:235]: Not sure if we need it really because we have the pagination, so there should be the need of scrolling. The Problem is react-custom-scrollbars-2 come with some side effects, on solution I found is to remove it or to use simplebar

Comment (florianjs) [reply lib/components/Table.tsx:235]: [Uploading Bildschirmaufzeichnung vom 2023-09-02, 12-51-29.webm…]()

**Review by DasProffi — DISMISSED** [review]

**Review by DasProffi — APPROVED** [review]
